### DISCUSSION TITLE
How can I omit the close button in Modals?

### DISCUSSION THREAD

**Discussion by ConsuelodeIruarrizaga** [twilio-labs/paste#2613] [Q&A] [answered]
Hi everyone, how are you?
I have doubts about modals. My front-end developer told me that he couldn't erase the cross button from the modal because of PASTE guidelines. Is it true? And if yes, why?

In this case, we have a modal with two buttons at the footer, and we don't want to give the user more options, and the cross doesn't provide them with enough information since it is a sensitive process. 
Here you can find an image,

<img width="758" alt="Screenshot 2022-08-17 at 18 50 26" src="https://user-images.githubusercontent.com/101562667/185198391-99fd5cdf-531c-4b49-8f96-461670b03797.png">

Thank you!!!!!
Consuelo

**Comment by kelseycavitt** [comment]
Hi @ConsuelodeIruarrizaga!

I think it may be helpful for me to gain more context of the workflow here, would you be able to send more screenshots or a link to a Figma file?

Modals need to allow the user the ability to close or "escape." To discourage a user from exiting the workflow, you could use an [alert dialog](https://paste.twilio.design/components/alert-dialog) if they do click the "close" button to confirm if they would like to leave the flow. 

Let us know if you more questions. I do think it would be helpful to see the flow too to help better understand the context! Thanks!

**Comment by TheSisb** [comment, marked as answer]
Hi @ConsuelodeIruarrizaga,

The Paste Modal is composable, which means that you can mix or omit certain components however you like. Our `ModalHeader` Component comes predefined with the close button for the reason Kelsey provided above. If you're certain that your experience does not need the close button (like really really certain), you can simply omit the `ModalHeader` in your composition and roll your own.
